package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "@guardian/image-rendering",
-  "version": "6.0.0",
+  "version": "6.1.0",
```

Comment:
This should probably be `7.0.0` due to https://github.com/guardian/types/pull/83 ?

Don't forget to bump the `version` field in `package-lock.json` too!

Reply:
good shout 👍